eth_utils/decorators.py

```diff
@@ -11,7 +11,7 @@ class combomethod(object):
     def __init__(self, method: Callable[..., Any]) -> None:
         self.method = method
 
-    def __get__(self, obj: T = None, objtype: Type[T] = None) -> Callable[..., T]:
+    def __get__(self, obj: T = None, objtype: Type[T] = None) -> Callable[..., Any]:
```

Comment:
@pipermerriam @cburgdorf This type hint was buggy. But it seems to me like an `Any` might be appropriate here since a method with the `combomethod` decorator can return any value - unless I'm missing any possible way to preserve the type of what the decorated function returns?

Reply:
I think to get the typing right this would need to be a `Generic` but it doesn't seem like it would play nicely with typing...

```python
class combomethod(Generic[T]):
    ...
```